llarp/router/rc_lookup_handler.cpp

```diff
@@ -288,7 +288,10 @@ namespace llarp
         GetRC(r, nullptr, true);
       return;
     }
-    // TODO: only explore via random subset
+    // service nodes gossip, not explore
+    if(_dht->impl->GetRouter()->IsServiceNode())
```

Comment:
Isn't `IsServiceNode()` going to be the same as `useWhiteList`? Which would make this a noop since (1) we hit the if block above and (2) it `return`s at the end, right?

Reply:
no.

Reply:
(Follow up: we decided these are "independent" or "orthogonal")